src/react/databrowser/objects/__tests__/MetadataSearch.test.jsx

```diff
@@ -0,0 +1,88 @@
+import * as zenkoActions from '../../../actions/zenko';
+import { Hint, Hints, HintsTitle } from '../../../ui-elements/Input';
+import MetadataSearch, { METADATA_SEARCH_HINT_ITEMS } from '../MetadataSearch';
+import { BUCKET_NAME } from '../../../actions/__tests__/utils/testUtil';
+import React from 'react';
+import { SearchButton } from '../../../ui-elements/Table';
+import { reduxMount } from '../../../utils/test';
+
+describe('Metadata Search', () => {
+    const newSearchListingMock = jest.spyOn(zenkoActions, 'newSearchListing');
+
+    afterEach(() => {
+        jest.clearAllMocks();
+    });
+
+    it('should render MetadataSearch component', () => {
+        const { component } = reduxMount(<MetadataSearch isMetadataType={false} bucketName={BUCKET_NAME}
+            prefixWithSlash='' errorZenkoMsg={null}/>);
+
+        expect(component.find(MetadataSearch).isEmptyRender()).toBe(false);
+    });
+
+    it('should render search button disabled by default', () => {
+        const { component } = reduxMount(<MetadataSearch isMetadataType={false} bucketName={BUCKET_NAME}
+            prefixWithSlash='' errorZenkoMsg={null}/>);
+
+        const button = component.find(SearchButton);
+        expect(button).toHaveLength(1);
+        expect(button.prop('disabled')).toBe(true);
+    });
+
+    it('should render dropdown menu correctly', () => {
+        const { component } = reduxMount(<MetadataSearch isMetadataType={false} bucketName={BUCKET_NAME}
+            prefixWithSlash='' errorZenkoMsg={null}/>);
+
+        const input = component.find('input');
+        expect(input).toHaveLength(1);
+
+        // check <Hint> are not rendered yet
+        expect(component.find(Hints)).toHaveLength(0);
+
+        // open dropdown menu
+        input.simulate('click');
+
+        // check <HintTitle> and <Hint> items are now rendered
+        expect(component.find(Hints)).toHaveLength(1);
+        expect(component.find(HintsTitle)).toHaveLength(1);
+        expect(component.find(HintsTitle).text()).toBe(' Suggestions ');
```

Comment:
Can you remove the extra spaces and use `toContain()` instead of `toBe()`

Reply:
done